packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -1,151 +1,147 @@
-describe('Abstract Modal', () => {
-  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+const selector = '#root';
 
-  describe('Append Class', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-append-class'));
+Terra.describeViewports('Abstract Modal', ['medium'], () => {
+  describe('Default Modal', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/default-abstract-modal'));
 
-    Terra.it.validatesElement({ selector: 'div[role="dialog"]' });
-  });
-
-  describe('Content Overflow', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-content-overflow');
-      browser.setViewportSize({ height: 1684, width: 1870 });
+    it('clicks to open modal', () => {
+      browser.click('button');
+      Terra.validates.element('open', { selector });
     });
 
-    after(() => browser.setViewportSize(Terra.viewports('medium')[0]));
-
-    Terra.it.validatesElement({ selector: 'div[role="dialog"]' });
-  });
-
-  describe('Custom Props', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-custom-props'));
-    it('checks for custom id prop', () => {
-      expect(browser.getAttribute('div[role="dialog"]', 'id')).to.equal('custom-props');
+    it('clicks outside to close modal', () => {
+      browser.moveToObject('[class*="ModalOverlay"]', 235, 5)
+        .leftClick();
+      Terra.validates.element('closed');
+      expect(browser.hasFocus('#modal-open-button')).to.be.true;
     });
-  });
 
-  describe('Disable Close On Esc', () => {
-    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-disable-close-on-esc'));
-
-    Terra.it.validatesElement('modal present', { selector: 'div[role="dialog"]' });
+    it('reopens the modal', () => {
+      browser.keys('Enter');
```

Comment:
We should test that the modal opened so that in the next test we know that if the modal is not open it was the escape key that closed it.

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/723/commits/ab686cd3944fc4ac7fd14e94b77e79a183debbcb